rdk-apps.xml

```diff
@@ -43,8 +43,8 @@
     <project groups="layers" name="meta-middleware-release" path="rdke/middleware/meta-middleware-release" remote="rdke" revision="refs/tags/2.1.2">
         <annotation name="MANIFEST_EXPORT_PATH" value="MANIFEST_PATH_MW_RELEASE"/>
     </project>
-
-    <project groups="layers" name="meta-application-rdke-dev" path="rdke/application/meta-application-rdke-dev" remote="rdkcentral" revision="refs/tags/4.0.0">
+    <!-- Revlock app components to last known good, taken from branch: feature/hotfix-rev-lock-refui -->
+    <project groups="layers" name="meta-application-rdke-dev" path="rdke/application/meta-application-rdke-dev" remote="rdkcentral" revision="2519a36be5999666fe22d9eae9e6414b243805ff">
```

Comment:
I think "SRCREV = "${AUTOREV}"" should also be deleted from both recipes. It is misleading.

This is on a feature branch. Shouldn't this be on a release branch instead, e.g. release/4.0.0-hotfix
fearture branches can be deleted without notice.

Also a tag, such as 4.0.1, would be better than a SHA1

Also add an `upstream` field to show which branch the commit is on. Just help with readability